ember_debug/object-inspector.js

```diff
@@ -993,7 +993,7 @@ function isInternalProperty(property) {
     '__LEGACY_OWNER',
     '__ARGS__',
     '__HAS_BLOCK__',
-    '__PROPERTY_DID_CHANGE__',
+    '-property_DID_CHANGE__',
```

Comment:
this doesn't look correct?

Reply:
Ah, yes good catch! I did a lot of find/replace and I guess I messed up some of that.